src/addListeners.js

```diff
@@ -5,7 +5,11 @@ var validate = require('./validate');
 var getErrorElement = require('./helpers/getErrorElement');
 
 // Handle elements with data-validations properties
-$(document).on('keyup click change blur', 'input, textarea, select', function (e) {
+$(document).on('keyup click change blur', 'input, textarea, select', onChange);
+$(document).on('blur keyup', '[data-validations]', onChange);
```

Comment:
Can this not just be `$(document).on('keyup click change blur', 'input, textarea, select, [data-validations]', onChange);`